pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "temporalio"
 version = "1.14.1"
 description = "Temporal.io Python SDK"
 authors = [{ name = "Temporal Technologies Inc", email = "sdk@temporal.io" }]
-requires-python = "~=3.9"
+requires-python = ">=3.9, <=3.13"
```

Comment:
Do we want to disallow 3.14? I would like to leave max open ended and if we are concerned about next Python, we solve that (I know we did some stuff for #863)

Reply:
Yes you're right, there was no reason to change the semantics; just the syntax. Although I have now changed semantics to allow Python 4 as you propose.